docs/FAQ.md

```diff
@@ -27,9 +27,9 @@ If it is a critical issue, identify first what error you have:
 
 Depending on the answers, while opening your issue, feel free to ping (just mention them with the arobase (@) symbol) appropriately so we can attempt to solve your problem faster:
 
-* [@guolinke](https://github.com/guolinke) (C++ code / R package / Python package)
-* [@Laurae2](https://github.com/Laurae2) (R package)
-* [@wxchan](https://github.com/wxchan) (Python package)
+* [@guolinke](https://github.com/guolinke) (C++ code / R-package / Python-package)
+* [@Laurae2](https://github.com/Laurae2) (R-package)
+* [@wxchan](https://github.com/wxchan) (Python-package)
```

Comment:
I think you also should be here 😄

Reply:
Thanks :smiley:

Reply:
BTW, why you aren't there?

Reply:
I don't know, but I will be glad if you add me in this PR :)

Reply:
Sure! I'll definitely add you in this PR.
>C++ code / Python-package?

Reply:
Just Python-package, please
because I don't understand over half of cpp codes :p

Reply:
OK 😄